keystone/api/projects.py

```diff
@@ -47,6 +48,49 @@ def _build_project_target_enforcement():
     return target
 
 
+def is_provider_tag(tag: str) -> bool:
+    """Check if the provided tag is a provider tag."""
+    return tag.startswith(tuple(CONF.provider_tag_prefix))
```

Comment:
Are you sure that this works? I would expect that something like this would be needed:
```
for prefix in CONF.provider_tag_prefix:
    if tag.startswith(prefix)
        return True
return False